src/main/java/org/storage/metadata/model/Role.java

```diff
@@ -0,0 +1,19 @@
+package org.storage.metadata.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Setter
+@Getter
```

Comment:
Lombok's @Data annotation will add getter, setter, constructor, tostring and equalas and hashcode